src/textual/css/styles.py

```diff
@@ -70,7 +70,7 @@
 class RulesMap(TypedDict, total=False):
     """A typed dict for CSS rules.
 
-    Any key may be absent, indiciating that rule has not been set.
+    Any key may be absent, indicating that rule has not been set.
```

Comment:
[while I was there] As I was digging into the code of Textual I noticed this typo, highlighted by PyCharm